### DISCUSSION TITLE
正常科学上网情况下无法完成购买

### DISCUSSION THREAD

**Discussion by Arlia-Etoile** [MaterialCleaner/MaterialCleaner#75] [Q&A]
Describe the bug

提示无法连接服务器
Response Code: BILLING_UNAVAILABLE
Debug Message: Google Play In-app Biling API version is less than 3

To Reproduce
正常科学上网情况下无法购买

Screenshots
[](https://user-images.githubusercontent.com/70079806/243325650-5367e87e-b5b7-4695-81dc-ff57b414aa8b.jpg)

Running environment (please complete the following information):

Device: Samsung S20
OS: Android 13 OneUI 5.1
Cleaner Version [1.10.2]
Cleaner Hooks Version [1.4.7]
MPM Version [Not Installed]

**Comment by GuhDoy** [comment]
感谢您的问题。

包含 Response Code 与 Debug Message 的错误提示都是由 [Play 结算库](https://developer.android.com/reference/com/android/billingclient/classes) 给出的，它们的出现通常与您的运行环境有关。

您可以参考 [开发者文档](https://developer.android.com/google/play/billing/errors#billing_unavailable_error_code_3) 中可能的解决方案来排查问题，对于已 root 的设备，可能还有更多原因造成此类问题，您可能还需要检查以下项目：

- 是否有应用或模块阻止了 Play 商店自启动。
- 是否有应用或模块阻止了 Cleaner 应用与 Play 商店交互。

**Reply by Arlia-Etoile** [reply]
已查明原因，需在“隐藏应用列表”中关闭对Google Play的隐藏

**Comment by GuhDoy** [comment]
得知您的问题已经解决，我们非常高兴！

感谢您支持 Cleaner！
